csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/source.extension.vsixmanifest

```diff
@@ -6,7 +6,8 @@
     <Description xml:space="preserve">This is a sample diagnostic extension for the .NET Compiler Platform ("Roslyn").</Description>
   </Metadata>
   <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0, 17.0)" />
```

Comment:
Indentation is off, and we should probably leave this one at `[15.0,)`. iirc that should be an open-ended range and not block 16.0.